Source/WebCore/dom/Node.cpp

```diff
@@ -118,11 +118,13 @@ struct SameSizeAsNode : EventTarget, CanMakeCheckedPtr<SameSizeAsNode> {
 #endif
     uint32_t refCountAndParentBit;
     uint32_t nodeFlags;
+    uint16_t elementStateFlags;
+    uint16_t styleBitfields;
```

Comment:
I don't see these used anywhere else.  Are they needed?

Reply:
See previous comment - these specifically reflect the `OptionSet<ElementStateFlag>` and `StyleBitfields` unpacked from the previous compact tuples in `Node`.